.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           - "24.04.2"
           - "latest-everything"
         shard: [1, 2, 3, 4, 5]
+        filter: [process, workflow]
```

Comment:
The pipeline-level tests are not included?

Reply:
ah good catch! should be pipeline and workflow

Reply:
And process? I think the idea is to test everything?

Reply:
I think its gonna overload the CI runners with too many tests to run

Reply:
I think we have to bank on the fact that the modules were/are tested in the modules repo and the pipeline CI runs only pipeline and workflow tests

Reply:
you should bring this up in the `#nf-test` channel and we can discuss/vote there

Reply:
What if there are local modules with tests? Moreover, the tests are only triggered for the changed modules.

If there are no local modules or functions with tests, let's skip `process`.

One more thing. If the nextflow.config changes or something from the `assets` directory changes, will the pipeline-level tests be triggered? I like the CI of nf-core/rnaseq which checks for these changes.

Reply:
plz bring this up in the `nf-test` channel and lets discuss there and get some consensus